app/mailers/patient_mailer.rb

```diff
@@ -110,8 +134,8 @@ def assessment_sms(patient)
                max_retries_message: I18n.t('assessments.sms.prompt.max_retries_message', locale: lang),
                thanks: I18n.t('assessments.sms.prompt.thanks', locale: lang) }
 
-    patient.update(last_assessment_reminder_sent: DateTime.now) # Update last send attempt timestamp before Twilio call
-    add_success_history(patient) if TwilioSender.start_studio_flow_assessment(patient, params)
+    patient.update(last_assessment_reminder_sent: DateTime.now) # Update last send attempt timestamp before Twilio sms assessment
+    patient.active_dependents_and_self.uniq.collect { |pat| add_success_history(pat) } if TwilioSender.start_studio_flow_assessment(patient, params)
```

Comment:
The additional `uniq` call here (and in similar lines below) shouldn't be necessary.